src/platform-aws.cpp

```diff
@@ -109,22 +111,42 @@ static struct ec2_platform_data platform_data_map[] = {
 		.gdr_required = false,
 		.default_protocol = PROTOCOL::SENDRECV,
 		.domain_per_thread = false,
+		.skip_rp_nics = false,
 		.env = {},
 	},
+	{
+		.name = "p6e-gb200.36xlarge",
```

Comment:
I'm ok not creating a separate entry for GB200 and just having a strncmp() in the init() function.  Probably less messy than having to maintain that regex below.